yarn.lock

```diff
@@ -1467,7 +1467,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/runtime-corejs2@npm:^7.0.0, @babel/runtime-corejs2@npm:^7.19.2":
+"@babel/runtime-corejs2@npm:^7.0.0":
```

Comment:
This dependency will go away entirely if you update the ud dependency.

Reply:
[d4a8d8e](https://github.com/InboxSDK/InboxSDK/pull/836/commits/d4a8d8e9e19ce9ef2024e94d7e4e535496de43da) did this with a 

```
yarn up ud
yarn up -R ud
```